### PR TITLE
OP-621 Settings->Types->Other Prices and New/Edit

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-621

### CODE REVIEW

File: src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java [hunk 1]
```diff
@@ -46,51 +44,51 @@
 import org.isf.utils.exception.gui.OHServiceExceptionUtil;
 import org.isf.utils.jobjects.MessageDialog;
 import org.isf.utils.jobjects.VoLimitedTextField;
+import org.isf.utils.layout.SpringUtilities;
 
 public class PricesOthersEdit extends JDialog {
 	
-	private EventListenerList OthersListeners = new EventListenerList();
+	private EventListenerList othersListeners = new EventListenerList();
```
Comment (mwithi) [inline src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java:51]: Might we refactor into `PricesOthersListeners` ?

**Review by mwithi — APPROVED** [review]
LGTM. Just one comment

File: src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java:51]: Yes we can.  Note it should start with a lowercase character.  I went ahead and changed method names to be more clear.

Comment (mwithi) [reply src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java]: sure, thanks!

**Pull request merged**